velox/type/Timestamp.h

```diff
@@ -178,6 +178,29 @@ struct Timestamp {
     return oss.str();
   }
 
+  std::string toStringTimeWithTimezone() const {
```

Comment:
could we refactor this code in a way that we don't need to duplicate the function above?

Reply:
Yeah, it would be sufficient to add an optional parameter to the above function to print timezone, and then invoke it here with nanoprecision and print timezone for the current behavior.

Reply:
@aditi-pandit @pedroerp I have reimplemented toString() to also handle the time with timezone format, please let me know if I need to change anything, thanks!